packages/frontend-plugin-api/src/analytics/useAnalytics.tsx

```diff
@@ -23,8 +23,11 @@ import { Tracker } from './Tracker';
 function useAnalyticsApi(): AnalyticsApi {
   try {
     return useApi(analyticsApiRef);
-  } catch {
-    return { captureEvent: () => {} };
+  } catch (error) {
+    if (error.name === 'NotImplementedError') {
+      return { captureEvent: () => {} };
+    }
+    throw error;
```

Comment:
While we were testing, we realized that we forgot the required settings in the `app-config.yaml` file, so the module implementation was never used in the tracker and it took a while to figure out why because the missing configuration error was suppressed by this try-catch. So we were not getting errors at all for the missing configs.

Reply:
Aaaah, nice.  That makes much more sense in the new frontend world, then.